src/modules/views/TaskView.jsx

```diff
@@ -89,32 +89,38 @@ const TaskView = ({ id }) => {
   };
 
   const checkReady = (trueFalse, updatedTask) => {
+     console.log("updatedTask!:", updatedTask)
     if (taskComplete !== trueFalse) {
-      setTaskComplete(trueFalse);
+      setTaskComplete(trueFalse)
+      
       setUpdatedTask(updatedTask)
+     
     }
   };
     
   const handleClick = async () => {
     // only allowed to click when requirements have been met
       // taskComplete is already true
-    const taskUpdates = {
-      "is_completed": true,
-      "message": updatedTask.message || null,
-      "image": updatedTask.image || null,
-    }  
-    // make API POST with updatedTask state
-    await updateSelectedTaskAPI(id, taskUpdates)
-    // clears selected task data to null
-    addTaskToState("selectedTask", {})
-    // redirect to missionView
+    
+      let data = new FormData();
```

Comment:
sooooo tasty! Who knew the content type of the API call would shake things up so much?